zeppelin-zengine/pom.xml

```diff
@@ -39,6 +39,7 @@
     <hadoop.version>2.7.3</hadoop.version>
     <commons.lang3.version>3.4</commons.lang3.version>
     <commons.vfs2.version>2.0</commons.vfs2.version>
+    <gcs.storage.version>1.14.0</gcs.storage.version>
```

Comment:
is this a new dependency? let's add that to the LICENSE file

Reply:
Done